hermes/crates/cardano-chain-follower/src/multi_era_block_data.rs

```diff
@@ -106,7 +106,7 @@ impl MultiEraBlock {
                 ));
             }
         } else {
-            if *previous > slot {
+            if *previous >= slot {
                 return Err(Error::Codec(
                     "Previous slot is not less than current slot".to_string(),
```

Comment:
```suggestion
                    "Previous slot is not less or equal than current slot".to_string(),
```

Reply:
Yes, this should be a >= check.
Good catch.